lib/src/dbs/executor.rs

```diff
@@ -140,9 +159,15 @@ impl<'a> Executor<'a> {
 	pub async fn execute(
 		&mut self,
 		mut ctx: Context<'_>,
-		mut opt: Options,
+		opt: Options,
 		qry: Query,
 	) -> Result<Vec<Response>, Error> {
+		// Take the notification channel
+		let chn = opt.sender.clone();
+		// Create a notification channel
+		let (send, recv) = channel::unbounded();
```

Comment:
Unbounded channels are a little bit suspicious. Is there a clear reason why the internal queue won't expand indefinitely i.e. in the event that the receiver stops working?

Reply:
They absolutely are. We decided to use unbounded for now, and will add a configuration later for it.
The main reason is that we don't have a ballpark number. It could, hypothetically, be quite big if there are many live queries, many updates, and sending down websockets is slow.
This was agreed with Tobie already. Additionally, this particular change is part of #2136